packages/yourloops/test/integration/pages/verify-email/verify-email-page.spec.tsx

```diff
@@ -0,0 +1,81 @@
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import * as auth0Mock from '@auth0/auth0-react'
+import { getAccessTokenWithPopupMock, logoutMock } from '../../mock/auth0.hook.mock'
+import { renderPage } from '../../utils/render'
+import { screen, waitFor, within } from '@testing-library/react'
+import { checkFooter } from '../../assert/footer'
+import userEvent from '@testing-library/user-event'
+import { AUTH0_ERROR_EMAIL_NOT_VERIFIED } from '../../../../lib/auth/models/auth0-error.model'
+
+describe('Verify email page', () => {
+  it('should display a description of the email verification process with options', async () => {
+    (auth0Mock.useAuth0 as jest.Mock).mockReturnValue({
+      isAuthenticated: false,
+      isLoading: false,
+      user: null,
+      getAccessTokenWithPopup: getAccessTokenWithPopupMock,
+      logout: logoutMock,
+      getAccessTokenSilently: jest.fn().mockRejectedValue({ error_description: AUTH0_ERROR_EMAIL_NOT_VERIFIED })
+    })
+
+    const router = renderPage('/verify-email')
+    await waitFor(() => {
+      expect(router.state.location.pathname).toEqual('/verify-email')
+    })
+
+    checkFooter({ needFooterLanguageSelector: true })
+
+    const pageHeader = within(screen.getByTestId('verify-email-header'))
+    expect(pageHeader.getByText('Logout')).toBeEnabled()
+    expect(pageHeader.getByLabelText('YourLoops Logo')).toBeVisible()
+
+    const pageContent = within(screen.getByTestId('verify-email-content'))
+    expect(pageContent.getByLabelText('YourLoops Logo')).toBeVisible()
+    expect(pageContent.getByText('Verify your email address')).toBeVisible()
+    expect(pageContent.getByText('To use YourLoops, you need to confirm your email address by clicking on the link we sent you. This helps to keep your account secure.')).toBeVisible()
+
+    const contactSupportSection = pageContent.getByTestId('verify-email-details-2')
+    expect(contactSupportSection).toHaveTextContent('No mail in your inbox or spam folder? Contact the customer support.')
+    expect(within(contactSupportSection).getByRole('button', { name: 'Contact the customer support' })).toBeVisible()
+
+    const createNewAccountSection = pageContent.getByTestId('verify-email-details-3')
+    expect(createNewAccountSection).toHaveTextContent('Wrong address? If you mistyped your email when signing up, create a new account.')
+    expect(within(createNewAccountSection).getByRole('button', { name: 'create a new account' })).toBeVisible()
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches to further communicate your intent.  -->
**🎯 suggestion (non-blocking):** ‏Links click could be tested too 😉
```suggestion
    const createNewAccountButton = within(createNewAccountSection).getByRole('button', { name: 'create a new account' })
    expect(createNewAccountButton).toBeVisible()
    await userEvent.click(createNewAccountButton)
    expect(logoutMock).toHaveBeenCalledTimes(1)
    expect(loginWithRedirectMock).toHaveBeenCalledTimes(1)
```